lib/dish.rb

```diff
@@ -0,0 +1,28 @@
+class Dish
+
```

Comment:
Watch your indentation - set your Sublime tabs to 2 spaces (the Sublime pill explains how to do that).